config/jobs/kubernetes-sigs/cluster-api/cluster-api-main-periodics-upgrades.yaml

```
@@ -44,11 +44,11 @@ periodics:
         privileged: true
       resources:
         requests:
-          cpu: 7300m
-          memory: 32Gi
+          cpu: 4600
```

I think 4600 is a bit high, right? 😅 

```suggestion
          cpu: 4600m
```